### ISSUE TITLE
When reopening tickets ask users to edit deadline

### ISSUE THREAD

**Issue by lejlac** [occrp/id-backend#129]
Now when users reopen old tickets the deadline stays the same which makes it look like the researcher has missed the deadline by months sometimes.
Can we find a way to prompt the requester to edit the deadline when they reopen?

**Comment by sungraze** [comment]
Yes, I can add a deadline field after the comment field where the user can change it. I could also mark it red if the deadline has passed so it's more obvious.

@stas you ok with this? you'd have to check the deadline attr as well when the status is changed to `in-progress` on PATCH.

**Comment by stas** [comment]
@sungraze I think we're good atm, we update the deadline independently from the status or reopen reason. Feel free to update it :+1:
